### PR TITLE
5.4 compatibility

### PR DESCRIPTION
**Pull request by nferrar-modio**

### CODE REVIEW

File: Config/DefaultGame.ini [hunk 1]
```diff
@@ -4,10 +4,10 @@
 ProjectID=0DA629C640F9BB551736EC95C1359C90
 Description=Memory game as mod.io demo
 CopyrightNotice=Copyright (c) 2023 mod.io
-CompanyName=mod.io
+CompanyName=
```
Comment (originalusername63) [inline Config/DefaultGame.ini:7]: Out of curiosity why did you remove the company name?

Comment (nferrar-modio) [reply Config/DefaultGame.ini:7]: Some platforms were failing to compile due to that name not meeting format requirements (tried `modio` too), I suspect it might be to do with developer ID or something, judging by the error in question.

No real reason we can't re-add it though, as I realise this doesn't *need* to compile for consoles.

Comment (originalusername63) [reply Config/DefaultGame.ini:7]: Ah ok, all good was just curious. LGTM

**Review by originalusername63 — APPROVED** [review]

**Comment by BenDawes** [comment]
@originalusername63  I wasn't able to build the project in ue 5.3 without changing 

`#include "AssetData.h"`

to
```cpp
	#if ENGINE_MAJOR_VERSION < 5
	#include "AssetData.h"
	#else
	#include "AssetRegistry/AssetData.h"
	#endif
```
and similarly for AssetRegistry.h

I thought maybe that should be included in this PR?

**Comment by gtreshchev** [comment]
> @originalusername63 I wasn't able to build the project in ue 5.3 without changing
> 
> `#include "AssetData.h"`
> 
> to
> 
> ```c++
> 	#if ENGINE_MAJOR_VERSION < 5
> 	#include "AssetData.h"
> 	#else
> 	#include "AssetRegistry/AssetData.h"
> 	#endif
> ```
> 
> and similarly for AssetRegistry.h
> 
> I thought maybe that should be included in this PR?

This is because the Modio plugin submodule is outdated, but it will be updated soon to support UE 5.3 and 5.4.
